plugins/inputs/odyssey/README.md

```diff
@@ -0,0 +1,75 @@
+### Configuration example
+
```

Comment:
Could you please add a short description what this input is about and what it is supposed to do. Furthermore, for people not knowing what _odyssey_ is about, it would help to add a reference or link to the project.